catalog/transformer/catalog-transformer-xml/pom.xml

```diff
@@ -165,17 +165,17 @@
                                         <limit implementation="org.codice.jacoco.LenientLimit">
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.50</minimum>
+                                            <minimum>0.61</minimum>
```

Comment:
❓Did the new tests increase the coverage?

Reply:
Nope 😄